python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -102,7 +105,7 @@ def fetchReferences(self, dataRef, exposure):
     def _makeArgumentParser(cls):
         parser = lsst.pipe.base.ArgumentParser(name=cls._DefaultName)
         parser.add_id_argument("--id", "deepCoadd_forced_src", help="data ID, with raw CCD keys + tract",
-                               ContainerClass=lsst.coadd.utils.CoaddDataIdContainer)
+                               ContainerClass=lsst.coadd.utils.ExistingCoaddDataIdContainer)
```

Comment:
@TallJimbo , did you want to me revert to CoaddDataIdContainer in multiBand.py as well (you didn't comment on it there, so I just want to make sure this was a global comment on this functionality).